GeneticsCore/resources/queries/geneticscore/mhc_data_source.sql

```diff
@@ -0,0 +1,41 @@
+SELECT
+  s.subjectId as Id,
+  s.allele,
+  --s.primerPairs,
+  s.shortName,
+  s.totalRecords as testsPerformed,
```

Comment:
```suggestion
  s.totalRecords as totalTests,
```

Reply:
@bbimber This looks like the right change to me to get things aligned between the SQL and XML, but can you confirm?